### DISCUSSION TITLE
[Announcement] The name of the repository of HW5

### DISCUSSION THREAD

**Discussion by goodtaeeun** [prosyslab-classroom/cs524-program-analysis#265] [Announcements]
For those of you early birds who checked out the homework repository,
if your repository has the name "cs492-2023-fall-hw5-*" please delete it and regenerate a new one with the same link.

I have updated the naming convention.
Sorry for the inconvenience.

**Comment by 33577** [comment]
Hi :) 
I think that we can't delete the repository. There is no permission to access setting page of the repository.

**Comment by goodtaeeun** [comment]
I see, then can you try to re-accept the link?
If that does not work, I will remake the assignment itself.

**Comment by m-spitfire** [comment]
It doesn't work as well. Clicking link results in saying I've already accepted assignment.

**Comment by goodtaeeun** [comment]
Okay. I have regenerated the assignment.
Please reaccept it
